packages/fiori/src/NavigationLayout.ts

```diff
@@ -69,27 +85,12 @@ class NavigationLayout extends UI5Element {
 	isTablet = isTablet() && !isCombi();
 
 	/**
-	 * Indicates whether the side navigation is collapsed.
-	 * @default false
+	 * Gets whether the side navigation is collapsed.
 	 * @public
+	 * @default false
 	 */
-	@property({ type: Boolean })
-	set sideCollapsed(value: boolean) {
-		this._sideCollapsed = value;
-
-		if (isPhone()) {
-			return;
-		}
-
-		const sideNavigation = this.sideContent[0];
-
-		if (sideNavigation) {
-			sideNavigation.collapsed = value;
-		}
-	}
-
-	get sideCollapsed() : boolean {
-		return this._sideCollapsed;
+	get effectiveMode() : boolean {
```

Comment:
I see few problems with this:
- if it is called effectiveMode it should return Collapsed or Expanded, if it is boolean it should be called otherwise - e.g. sideCollapsed or effectivelyCollapsed (or method isSideCollapsed or isEffectivelyCollapsed)
- there is no default value, or default value should be null
- at the moment this value will be wrong or not useful before the rendering
-- it can be calculated in the getter. This might be problematic if we make it depend on parent container width someday
-- or document that the value is unknown before the rendering and return null

so I suggest this:

```
/**
 * Gets the effective mode for the side content - "Collapsed" or "Expanded".
 * @public
 */
getEffectiveMode() : NavigationLayoutMode|null { // or checkEffectiveMode
	this.calcEffectiveMode(); // if we can not calculate someday - we return null
	return this._effectiveMode; // "Collapsed" or "Expanded"
}
```

or

```
/**
 * Is the side content is effectively collapsed.
 * @public
 */
isSideCollapsed() : boolean|null {
	this.calcEffectiveMode(); // if we can not calculate someday we return null
	return this._effectiveMode; // true or false
}
```

@vladitasev what do you think?